homeassistant/components/lupusec/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "lupusec",
   "name": "Lupus Electronics LUPUSEC",
   "documentation": "https://www.home-assistant.io/integrations/lupusec",
-  "requirements": ["lupupy==0.2.4"],
```

Comment:
I could not find the release or tag upstream?

Did something went wrong?

Reply:
Tag is now there